src/open_inwoner/accounts/tests/test_oidc_views.py

```diff
@@ -3,8 +3,12 @@
 from django.contrib.auth import get_user_model
 from django.test import TestCase
 from django.urls import reverse
+from django.utils.translation import gettext_lazy as _
```

Comment:
If we don't have module level translatables we can use the normal, non-lazy version.